failed to resolve instance reference for default export

Caused by:
    an import with name `a` does not exist